File: domain/src/main/java/com/aceplus/domain/model/forApi/VolumeDiscountItem.java
package com.aceplus.domain.model.forApi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by haker on 2/10/17.
 */
public class VolumeDiscountItem {
    @SerializedName("Id")
    @Expose
    private String id;
    @SerializedName("VolumeDiscountId")
    @Expose
    private String volumeDiscountId;
    @SerializedName("FromSaleAmt")
    @Expose
    private String fromSaleAmt;
    @SerializedName("ToSaleAmt")
    @Expose
    private String toSaleAmt;
    @SerializedName("DiscountPercent")
    @Expose
    private String discountPercent;
    @SerializedName("DiscountAmount")
    @Expose
    private Double discountAmount;
    @SerializedName("DiscountPrice")
    @Expose
    private Double discountPrice;
    @SerializedName("Active")
    @Expose
    private String active;
    @SerializedName("TS")
    @Expose
    private String tS;
    @SerializedName("CreatedDate")
    @Expose
    private String createdDate;
    @SerializedName("CreatedUserID")
    @Expose
    private String createdUserID;
    @SerializedName("UpdatedDate")
    @Expose
    private String updatedDate;
    @SerializedName("UpdatedUserID")
    @Expose
    private String updatedUserID;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getVolumeDiscountId() {
        return volumeDiscountId;
    }

    public void setVolumeDiscountId(String volumeDiscountId) {
        this.volumeDiscountId = volumeDiscountId;
    }

    public String getFromSaleAmt() {
        return fromSaleAmt;
    }

    public void setFromSaleAmt(String fromSaleAmt) {
        this.fromSaleAmt = fromSaleAmt;
    }

    public String getToSaleAmt() {
        return toSaleAmt;
    }

    public void setToSaleAmt(String toSaleAmt) {
        this.toSaleAmt = toSaleAmt;
    }

    public String getDiscountPercent() {
        return discountPercent;
    }

    public void setDiscountPercent(String discountPercent) {
        this.discountPercent = discountPercent;
    }

    public Double getDiscountAmount() {
        return discountAmount;
    }

    public void setDiscountAmount(Double discountAmount) {
        this.discountAmount = discountAmount;
    }

    public Double getDiscountPrice() {
        return discountPrice;
    }

    public void setDiscountPrice(Double discountPrice) {
        this.discountPrice = discountPrice;
    }

    public String getActive() {
        return active;
    }

    public void setActive(String active) {
        this.active = active;
    }

    public String getTS() {
        return tS;
    }

    public void setTS(String tS) {
        this.tS = tS;
    }

    public String getCreatedDate() {
        return createdDate;
    }

    public void setCreatedDate(String createdDate) {
        this.createdDate = createdDate;
    }

    public String getCreatedUserID() {
        return createdUserID;
    }

    public void setCreatedUserID(String createdUserID) {
        this.createdUserID = createdUserID;
    }

    public Object getUpdatedDate() {
        return updatedDate;
    }

    public void setUpdatedDate(String updatedDate) {
        this.updatedDate = updatedDate;
    }

    public Object getUpdatedUserID() {
        return updatedUserID;
    }

    public void setUpdatedUserID(String updatedUserID) {
        this.updatedUserID = updatedUserID;
    }
}

File: domain/src/main/java/com/aceplus/domain/entity/volumediscount/VolumeDiscountItem.kt
package com.aceplus.domain.entity.volumediscount

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


@Entity(tableName = "volume_discount_item")
class VolumeDiscountItem {

    @PrimaryKey
    @ColumnInfo(name = "id")
    @SerializedName("id")
    @Expose
    var id: Int = 0

    @ColumnInfo(name = "volume_discount_id")
    @SerializedName("volume_discount_id")
    @Expose
    var volume_discount_id: Int = 0

    @ColumnInfo(name = "from_sale_amount")
    @SerializedName("from_sale_amount")
    @Expose
    var from_sale_amount: Double = 0.0

    @ColumnInfo(name = "to_sale_amount")
    @SerializedName("to_sale_amount")
    @Expose
    var to_sale_amount: Double = 0.0

    @ColumnInfo(name = "discount_percent")
    @SerializedName("discount_percent")
    @Expose
    var discount_percent: Double = 0.0

    @ColumnInfo(name = "discount_amount")
    @SerializedName("discount_amount")
    @Expose
    var discount_amount: Double = 0.0

    @ColumnInfo(name = "discount_price")
    @SerializedName("discount_price")
    @Expose
    var discount_price: Double = 0.0

    @ColumnInfo(name = "active")
    @SerializedName("active")
    @Expose
    var active: String? = null

    @ColumnInfo(name = "created_date")
    @SerializedName("created_date")
    @Expose
    var created_date: String? = null

    @ColumnInfo(name = "created_user_id")
    @SerializedName("created_user_id")
    @Expose
    var created_user_id: Int = 0

    @ColumnInfo(name = "updated_date")
    @SerializedName("updated_date")
    @Expose
    var updated_date: String? = null

    @ColumnInfo(name = "updated_user_id")
    @SerializedName("updated_user_id")
    @Expose
    var updated_user_id: Int = 0

    @ColumnInfo(name = "timestamp")
    @SerializedName("timestamp")
    @Expose
    var timestamp: String? = null
}